### ISSUE TITLE
Terraform 0.3.7 bug in source/dest checking

### ISSUE THREAD

**Issue by DoctorVin** [airpair/ntiered-aws-docker-terraform-guide#1]
The current version of Terraform has a bug which doesn't set source_dest_check properly. This will interfere with the deploy of the web tier, because the gateway won't actually pass traffic from the private VPC side to the internet.

This can be worked around by delaying the deploy of the web tier, and manually setting source/dest. check in the AWS console for the NAT instance before continuing.

**Comment by gosuri** [comment]
I'm able to set `source_dest_check = false` on nat resource and it works okay. Can you please upload please upload your source so I can take look. 

I'm using `Terraform v0.3.7` on MacOS

**Comment by DoctorVin** [comment]
I can replicate this with a fresh pull of this from GitHub. Likewise I am using Terraform 0.3.7 on Mac OS X .

`$ terraform -version
Terraform v0.3.7`

I have a more-minimal test case that skips some of the private VPC stuff, and that gives identical results in my hands. 

I followed up with the folks at Hashicorp (because it looked like a Terraform bug) and they said this had been fixed on their master branch. 

https://github.com/hashicorp/terraform/issues/1160
